spec/models/event_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe Event do
+  describe '#search_by_name' do
```

Comment:
zamiast
`describe '#search_by_name' do`
powinno byzamiast
`describe '.search' do`

`.` zamiast `#`, ponieważ to jest metoda statyczna (Robimy `Event.search` a nie `Event.new.search`)